k8s/mongodb/container/README.md

```diff
@@ -0,0 +1,88 @@
+## Custom MongoDB container for BigchainDB Backend
```

Comment:
This was made so that people who do not want to use the cluster setup but still want to use our images locally for dev can use it.